mw-webapp/src/router/ProtectedRouter.tsx

```diff
@@ -0,0 +1,19 @@
+import {Navigate, Outlet, useLocation} from "react-router-dom";
+import {pages} from "src/router/pages";
+import {PATH, USER_IS_AUTH} from "src/service/auth/constants";
+
+/**
+ * Protected Router
+ */
+export const ProtectedRouterLayer = () => {
```

Comment:
Are you sure that it's a layer? If yes then need to change architecture.draw.io

Reply:
Can you suggest a better name?

Reply:
I don't know but it's not a layer